Jolt/Physics/Body/BodyManager.cpp

```diff
@@ -558,14 +559,14 @@ bool BodyManager::RestoreState(StateRecorder &inStream)
 }
 
 #ifdef JPH_DEBUG_RENDERER
-void BodyManager::Draw(const DrawSettings &inDrawSettings, const PhysicsSettings &inPhysicsSettings, DebugRenderer *inRenderer)
+void BodyManager::Draw(const DrawSettings &inDrawSettings, const PhysicsSettings &inPhysicsSettings, DebugRenderer *inRenderer, const BodyDrawFilter* inBodyFilter)
```

Comment:
Can you please write this as const BodyDrawFilter *inBodyFilter (shift space), I know it's weird, but it's unfortunately the style I adopted 10+ yrs ago.